server/datastore/mysql/migrations/tables/20240617174138_UpdateMobileConfigColumnType_test.go

```diff
@@ -0,0 +1,30 @@
+package tables
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240617174138(t *testing.T) {
+	db := applyUpToPrev(t)
+
+	// Create a 1mb long string. This will fail at first, but will work with new column type.
+	var b strings.Builder
+	b.Grow(1000000)
+	for i := 0; i < 1000000; i++ {
```

Comment:
That won't hit the DB limit of `MEDIUMBLOB` (which is 4GB, `2^32`). I think testing that you _can_ insert a 1MB profile is good enough, no need to trigger the limit.

Reply:
Oh sorry, that's the _before_ limit, so it will definitely hit it. Ignore that :D